res/database/Default/database_scripts.xml

```diff
@@ -141,28 +141,31 @@
 			</title>
 			<description>
 				<de>%JOB% %ROLENAME1% %ACTION% %ENEMY%. Sie wollen %VICTIM% %ACTION2%.</de>
-				<en>%JOB% %ROLENAME1% %ACTION% %ENEMY%. They want to %ACTION2%.</en>
+				<en>%JOB% %ROLENAME1% %ACTION% %ENEMY% who want to %ACTION2%.</en>
 			</description>
 
 			<variables>
 				<!-- international title for all incarnations - so "de" is enough -->
+				<!-- scr0llbaer: that may be true, but probably still not a good idea, because someone might add nested variables later, which then would resolve into German. -->
```

Comment:
Have you checked that indeed nested variables then use German rather than the original language? If this is the case, it is a bug in variable resolution.
If not necessary, the duplication should be avoided.

Reply:
Yes, tried and tested.
And wasn't even sure the way it's resolved can be considered a bug, but you decide.

Reply:
I was able to reproduce the problem and have proposed a fix. If it is working and accepted, duplicating identical titles/descriptions would not be necessary anymore.